spec/test.ts

```diff
@@ -0,0 +1,28 @@
+import { webcrypto } from "crypto";
+import "jest-preset-angular/setup-jest";
+
+Object.defineProperty(window, "CSS", { value: null });
```

Comment:
Could we create this polyfill by importing all `*/test.ts` instead? And avoid the duplicated code.

Reply:
I didn't do that because we don't need it in all of our jslib projects. I don't want web stuff defined in `node`, for example, because a lot of our prod code changes functionality depending on the existence of `window`

Reply:
I was more thinking of having the root `test.ts` import the children to avoid duplicating that logic.

Reply:
It's in `common`, any problem just moving it there?